0009-enpoint-events-ipc/diconnected.avsc

```diff
@@ -0,0 +1,41 @@
+{
+  "namespace": "org.kaaproject.ipc.event.gen.v1.endpoint.connectivity",
+  "type": "record",
+  "name": "EndpointDisconnectedEvent",
+  "doc": "Endpoint(s) disconnected event message.",
+  "fields": [
+    {
+      "name": "correlationId",
+      "type": "string",
+      "default": "",
+      "doc": "Message tracing ID primarily used for tracking the message processing across services."
+    },
+    {
+      "name": "timestamp",
+      "type": "long",
+      "doc": "Unix time in milliseconds when the message was created."
+    },
+    {
+      "name": "timeout",
+      "type": "long",
+      "default": -1,
+      "doc": "The amount of milliseconds since timestamp when the message expires at the originating entity."
+    },
+    {
+      "name": "endpointIds",
+      "doc": "Endpoint identifiers. May contain multiple items when client that was associated with multiple endpoints closed the communication session.",
+      "type": {
+        "type": "array",
+        "items": "string",
```

Comment:
@vtkhir As discussed offline, let's open a backlog ticket for adding the appVersionName to the array if/when needed in real world.